### ISSUE TITLE
Option to backup all namespaces at once

### ISSUE THREAD

**Issue by selivan** [selivan/kibana-backup-simple#6]
`--backup-all-spaces`

Check if it can still work with single file. If not - add options:

`--backup-file-prefix` - takes path, generates backups into `<backup-file-prefix>-<namespace_id>.ndjson`
`--backup-file` - for usual backup, if I started supporting something besides stdin/stdout

https://www.elastic.co/guide/en/kibana/current/spaces-api-get-all.html

**Comment by selivan** [comment]
```
curl -s -H 'Content-Type: application/json' -H 'kbn-xsrf: reporting' http://localhost:5601/api/spaces/space | jq .
```

**Comment by selivan** [comment]
Closed in 380a6f9
